migrations/20220509173028_create_addresses_table.sql

```diff
@@ -7,7 +7,7 @@ BEGIN;
 CREATE TABLE addresses(
 	id INT,
 	PRIMARY KEY (id),
-	user_id INT,
+	user_id INT NOT NULL,
```

Comment:
I am not sure if this modification will work during migration. In my experience I had to create a new migration to alter the column for a previously-ran migration. If you had no problems with this, then please ignore this comment.

Reply:
I did not experience issues running migrations with this change. However, if you do when you re-run the migration, please let me know so that I will add a new migration to alter the column. Thank you
<img width="947" alt="Screen Shot 2022-05-30 at 11 52 55 AM" src="https://user-images.githubusercontent.com/28492640/170977518-852a2462-bdd5-48b6-9644-2fbddde47425.png">
.